dev/Repeater/APITests/ViewManagerTests.cs

```diff
@@ -586,6 +586,43 @@ public void CanResetLayoutAfterUniqueIdReset()
             });
         }
 
+        [TestMethod]
+        public void ValidateDataContextDoesNotGetOverwritten()
+        {
+            ItemsRepeater repeater = null;
+            const string c_element1DataContext = "Element1_DataContext";
+
+            RunOnUIThread.Execute(() =>
+            {
+                List<Button> data = new List<Button>();
+
+                var element1 = new Button();
+                element1.Content = "Element1_Content";
+                element1.DataContext = c_element1DataContext;
+                data.Add(element1);
```

Comment:
nit: you can use initializer syntax here.
var data = new List<Button>()
{
    new Button()
    {
         Content = "Element1_Content",
          DataContext = c_element1DataContext
    }
}